R/telemetry.R

```diff
@@ -504,6 +440,7 @@ Telemetry <- R6::R6Class( # nolint object_name.
         navigation_inputs = c(),
         excluded_inputs_regex = excluded_inputs_regex,
         include_input_ids = include_input_ids,
+        track_errors = TRUE,
```

Comment:
Prevents javascript-based errors from being duplicated in the log and shouldn't be in public API.

This is a convenience pass-through used from `start_session` and relevant in versions before `shiny@1.8.1`

Reply:
Looks good to me